btcjson/chainsvrcmds.go

```diff
@@ -864,6 +873,7 @@ func init() {
 	MustRegisterCmd("gettxout", (*GetTxOutCmd)(nil), flags)
 	MustRegisterCmd("gettxoutproof", (*GetTxOutProofCmd)(nil), flags)
 	MustRegisterCmd("gettxoutsetinfo", (*GetTxOutSetInfoCmd)(nil), flags)
+	MustRegisterCmd("getzmqnotifications", (*GetZmqNotificationsCmd)(nil), flags)
```

Comment:
Wondering if `getzmqnotifications` is really a _chain_ command. Bitcoin Core categorizes it separately under the [`zmq` section](https://bitcoincore.org/en/doc/0.19.0/rpc/zmq/getzmqnotifications/). Maybe `walletsvrcmds.go` is a better file for this?

What do you think, @jcvernaleo @jakesyl?